### ISSUE TITLE
config/create-API interprets backslash-paths as escaping

### ISSUE THREAD

**Issue by alexgit2k** [rclone/rclone#6673]
#### What is the problem you are having with rclone?
https://rclone.org/local/#paths-on-windows says that the path separator can be \ or / on windows, which is not working at config/create. Reproduceable on Windows with:

1. rclone --rc-no-auth rcd
2. POST http://localhost:5572/config/create?name=Normal&type=local&parameters={}
3. POST http://localhost:5572/config/create?name=Crypted&type=crypt&parameters={"remote":"Normal:C:\path1\path2"}

```
{
	"error": "key \"parameters\": invalid character 'p' in string escape code",
	"input": {
		"name": "Crypted",
		"parameters": "{\"remote\":\"Normal:C:\\path1\\path2\"}",
		"type": "crypt"
	},
	"path": "config/create",
	"status": 400
}
```

It is working if the path is escaped or / is used instead:
- POST http://localhost:5572/config/create?name=Crypted&type=crypt&parameters={"remote":"Normal:C:\\path1\\path2"}
- POST http://localhost:5572/config/create?name=Crypted&type=crypt&parameters={"remote":"Normal:C:/path1/path2"}

mount/mount supports \:
- POST http://localhost:5572/mount/mount?mountPoint=X:&fs=Normal:C:\path1\path2

#### What is your rclone version (output from `rclone version`)
rclone v1.61.1

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
Windows 11, 64 bit

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
POST http://localhost:5572/config/create?name=Crypted&type=crypt&parameters={"remote":"Normal:C:\path1\path2"}

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
```
rclone -vv --rc-no-auth rcd
2023/01/06 17:10:10 DEBUG : rc: "config/create": with parameters map[name:Crypted parameters:{"remote":"Normal:C:\path1\path2"} type:crypt]
2023/01/06 17:10:10 ERROR : rc: "config/create": error: key "parameters": invalid character 'p' in string escape code
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
`\` is the escape character in JSON - you will need to double them up to `\\`.

All the input and output to the API is based on JSON strings.

**Comment by alexgit2k** [comment]
Thanks for the fast feedback.

But at the mount/mount-API `\` is working without problems. I also do not have any problems with `\` at the sync/sync-API.
Probably there is a special handling for `\` for `srcFs` and `dstFs`, but not for `remote` in `parameters`.

**Comment by ncw** [comment]
> But at the mount/mount-API `\` is working without problems. I also do not have any problems with `\` at the sync/sync-API. Probably there is a special handling for `\` for `srcFs` and `dstFs`, but not for `remote` in `parameters`.

Yes you are right. `srcFs` and `dstFs` will be parsed as remotes and the parser knows how to swap `\` for `/`. The `remote` parameter isn't parsed at all.

I think I don't want to change that, and would prefer to document it.

Would that cause you a problem? Would you like to contribute some docs? Probably in https://github.com/rclone/rclone/blob/master/docs/content/rc.md

**Comment by alexgit2k** [comment]
Thanks, I'll consider the handling for windows-remotes at `parameters` in my code.

Good that you could confirm this behavior. I will do a pull-request for the addition to the documentation.
